contents/founders/negotiate-software-better.md

```diff
@@ -0,0 +1,91 @@
+
+You know the feeling - you negotiate hard, the salesperson makes you feel really special, they really _get_ you, you’ve built a rapport, wait is this actually kinda fun?
+
+Then contract signed and boom - you’re handed over to another team where all of a sudden your questions take a few days to get a reply, not hours. It felt like they were bending over backwards to help at every stage, but now you’re left out there on your own. 
+
+This is down to the classic sales throws the customer over the wall to customer success/support once they’re done, and that new team’s incentives are very different - they basically need to do just enough to ensure that you stay. No extra special effort is needed any more now you’re locked in!
+
+This is why you need to agree _before_ you sign the contract:
+
+- Exactly what level of support you can expect. Negotiate what happens if they break this.
+- Who your ongoing point of contact will be - meet them first. Ask for someone else if they aren’t helpful.
+
+_Put it in writing_. If someone isn’t willing to put something in writing, they will definitely not do it. You are not friends. There is no social obligation to help you out here. 
+
+> The PostHog way: at PostHog, the [technical account executive](/handbook/growth/sales/overview#how-we-define-roles) who sells you the software is _also_ the person who manages your account in the long term. They’ll get to know you and your company really well, and are financially incentivized to make sure you _stay_ happy. We also tell you up front exactly what to expect from [onboarding](/handbook/growth/sales/customer-onboarding).
+
+# Avoid the nasties in the legal docs
+
+You’ve negotiated hard and agreed on a price you’re happy with - hooray! Wait, but it’s not quite over yet - here are a couple of nasty bits in contracts that Saas companies will include by default: 
+
+- ‘We will renew your contract each year by default unless you give us 90 days notice of your intention not to renew’ - take. this. out. Your salesperson will make a lot of noise about getting that secret approval we talked about before, but you do not want to rely on a random calendar reminder you set yourself 9 months from now. 
+  - You also want to give yourself an opportunity to renegotiate and improve your contract. 
+- You don’t have the right to terminate the contract, only they do. You want this to be mutual - you want to be able to terminate the contract if they breach the terms. In practice, performance is the number one thing you will probably want to terminate for, so make sure some kind of SLA is included.
+- Automatic price increase by X% at renewal - easiest way to avoid this is not having auto renewal in the first place. However, if they _really_ won’t budge (which can be the case in very large companies), make sure that they haven’t hidden something dumb like ‘inflation + 10%’. Reasonable here is sub-5%. 
```

Comment:
Styling + edited the second para slightly to match format of other points
```suggestion
- ‘We will renew your contract each year by default unless you give us 90 days notice of your intention not to renew’ - take. this. out. Your salesperson will make a lot of noise about getting that secret approval we talked about before, but you do not want to rely on a random calendar reminder you set yourself 9 months from now. 
  
  - You also want to give yourself an opportunity to renegotiate and improve your contract. 

- You don’t have the right to terminate the contract, only they do. - You want to be able to terminate the contract if they breach the terms, so this should be mutual. In practice, performance is the number one thing you will probably want to terminate for, so make sure some kind of SLA is included.

- Automatic price increase by X% at renewal - easiest way to avoid this is not having auto renewal in the first place. However, if they _really_ won’t budge (which can be the case in very large companies), make sure that they haven’t hidden something dumb like ‘inflation + 10%’. Reasonable here is sub-5%. 
```